R/resample_draws.R

```diff
@@ -0,0 +1,176 @@
+#' Resample Draws Objects
+#'
+#' Resample [`draws`] objects according to given weights obtained,
+#' for instance, through importance sampling.
+#'
+#' @template args-methods-x
+#' @param weights A vector of positive weights of length equal to the
+#' number of draws in `draws` (as obtained by method [`ndraws`]).
+#' Draws with higher weights will be selected more often, usually
```

Comment:
Is this sentence necessary? 
"Draws with higher weights will be selected more often, usually ... their weights."

Reply:
Probably not. I will remove it.